packages/neuron-wallet/tests/controllers/sync-api.test.ts

```diff
@@ -172,6 +172,7 @@ describe('SyncApiController', () => {
             cacheRate: undefined,
             estimate: undefined,
             status: 3,
+            validTarget: process.env.CKB_NODE_ASSUME_VALID_TARGET,
```

Comment:
I think we should avoid the side effects in unit tests when we can, and the `process.env` is a case of side effect, maybe we could launch the unit test with [--load-env-files=false](https://lerna.js.org/docs/features/run-tasks#automatic-loading-of-env-files) to disable it

Reply:
And the `env.CKB_NODE_ASSUMED_VALID_TARGET` can be mocked